### PR TITLE
Upgrade to GitLab 8.15.3

### PR DESCRIPTION
**Pull request by phenomax**
I just updated every version number, as we did it in previous versions.
Testing is highly appreciated!

### CODE REVIEW

**Comment by jostyee** [comment]
I wonder which update introduced this issue: https://github.com/sameersbn/docker-gitlab/issues/1036 seems just adjust version number is not enough.

**Comment by phenomax** [comment]
@jostyee Did you have a look, if this problem changed with this update?

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM sameersbn/ubuntu:14.04.20161217
```
Comment (kingcody) [inline Dockerfile]: @phenomax, the line was not redundant. There 'was' an invisible character (carriage return, I believe) leading the `FROM` instruction. The line should be added back, but with out the `^R`. You typically won't see this character on GitHub, but if you do a quick `git diff` you can tell it's there:
image

**Comment by sameersbn** [comment]
@phenomax please see this [comment](https://github.com/sameersbn/docker-gitlab/pull/1046#discussion_r95282335)

**Comment by phenomax** [comment]
@kingcody Thanks for the explanation.
Mine illegal character has been removed.
Can you please double check?

**Comment by Ilya-Kuchaev** [comment]
Any chance to see it merged?
8.15.4 has been released https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md#8154-2017-01-09

**Comment by kingcody** [comment]
@phenomax no problem, glad I could help. Indeed, the `FROM` instruction has been added back without the carriage return. Thank you for contributing this PR.

**Comment by phenomax** [comment]
xD love GitLab. While one PR ist being progressed, they just release another version

**Comment by solidnerd** [comment]
I will have look tomorrow in it and give you feedback.

**Comment by sameersbn** [comment]
Looks good to me..

**Pull request merged**